### DISCUSSION TITLE
Cache animated texture

### DISCUSSION THREAD

**Discussion by HenrikLinnes** [Autodesk/maya-usd#1921] [Q&A]
I have a plane with an image sequence attached to it, but when I cache it only uses the first frame of the image sequence. Is there a way to cache this correctly?

**Comment by JGamache-autodesk** [comment]
Hi @HenrikLinnes, 
I have attached a USD asset with animated texture. It does playback correctly in both usdView and MayaUSD. Cached playback also seems to work fine.
Can you give me more details on how to reproduce the issue you are having? Are you setting up your animation differently?
Thanks
[AnimatedTexture.zip](https://github.com/Autodesk/maya-usd/files/7926878/AnimatedTexture.zip)
